src/index.ts

```diff
@@ -173,14 +134,6 @@ export const REDUX_DEVTOOLS: ExtensionReference = {
   id: 'lmhkpmbekcpmknklioeibfkpmmfibljd',
   electron: '>=1.2.1',
```

Comment:
Would be clean to delete these `electron` properties and remove the check in `install.ts`

Reply:
Yeah that makes sense